demo/web/src/main/notebooks/A. Real-time table ops/A2 Filter and decorate .md

```diff
@@ -0,0 +1,69 @@
+# Filter and decorate
+
+Now that we have some data, let's work with it. In this notebook, we show how to decorate and filter our data.
+
+Let's start by creating some daily data. This could simulate something like daily stock prices or daily temperatures.
```

Comment:
```suggestion
Let's start by simulating a year's worth of daily measurements. This could simulate something like stock prices, temperatures, etc.
```